editor/src/app/sites/site.component.ts

```diff
@@ -140,4 +141,9 @@ export class SiteComponent implements OnInit {
       ],
     });
   }
+
+  updateSiteUrl(siteUrl) {
```

Comment:
The function name is not descriptive. The correct term is `navigate`, since we're not updating the URL of an existing site, we're navigating to the new site.